### ISSUE TITLE
Getting parsing errors on the Example Notion page 

### ISSUE THREAD

**Issue by Akash-Sharma-1** [leoncvlt/loconotion#63]
Hi,

I was trying to build files for this example [notion page](https://www.notion.so/Loconotion-Example-03c403f4fdc94cc1b315b9469a8950ef) given in the README itself. I received some parsing based errors on this notion page.

Heres's the error - stack trace : 
![image](https://user-images.githubusercontent.com/33986563/122576348-38edfc00-d06f-11eb-8424-8d8790641164.png)

I suspect it's related to the parsing of toggle buttons from the toggling list existing on the page.

**Comment by Akash-Sharma-1** [comment]
I was able to solve this, with few changes in the code of `notionparser.py`. 
Shall I put a pull request for this ? @leoncvlt

**Comment by leoncvlt** [comment]
Sounds good, what was the issue?

**Comment by Akash-Sharma-1** [comment]
Hi,
I now actually boiled it down to one-liner change.

Basically, the issue is at this [line](https://github.com/leoncvlt/loconotion/blob/b8a59cda17c8357cc1efe00c7f2a9a0e7f7a564d/loconotion/notionparser.py#L312), where it's trying to click the toggle button.

Here, my intuition is that it's not able to click the button (via the web driver) either without scrolling down or it's something to do with the inner toggle buttons (not very sure).
In such cases, I always prefer using javascript-based clicks over webdriver based clicks since webdriver clicks usually cause such problems (as webdriver tends to ensure that the element is visible in the view or is interactable otherwise it would throw errors).

Hence, I simply replaced this toggle button webdriver click to javascript based click and it seems to work now.
